static-assets-store/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM nginx:1.13
+
+RUN mkdir /assets && \
+    chmod 2777 /assets
```

Comment:
`2777`? For the sticky group?

Reply:
`article-store-airflow` writes as its `airflow` user. This is a bit brute-force, feel free to change but don't see it as important.

Reply:
777 is fine, is the `2` that sticks out, but not important at the moment